engines/duplicity.go

```diff
@@ -75,11 +75,12 @@ func (d *DuplicityEngine) Backup() (err error) {
 		return
 	}
 
-	if vol.Config.NoVerify {
-		log.WithFields(log.Fields{
-			"volume": vol.Name,
-		}).Info("Skipping verification")
-	} else {
+	verify, err := d.Handler.MustVerifyBackup(vol)
```

Comment:
Not a big fan of `Must`. Maybe `IsVerifyScheduled` or `IsCheckScheduled` and then call the boolean `checkScheduled`?